src/item.cpp

```diff
@@ -6945,42 +6970,28 @@ int item::ammo_remaining() const
     return 0;
 }
 
-int item::ammo_capacity() const
+int item::remaining_ammo_capacity() const
 {
-    return ammo_capacity( false );
+    if( ammo_types().empty() ) {
+        return 0;
+    }
+
+    const itype *loaded_ammo = ammo_data();
+    if( loaded_ammo == nullptr ) {
+        return ammo_capacity( item::find_type( ammo_default() )->ammo->type ) - ammo_remaining();
```

Comment:
This function is used in a couple places to ask "can this be reloaded?" and sometimes there's no ammo type available. I thought that it would make sense to have the default ammo as a fallback if there was no ammo loaded in order to calculate some number, instead of just returning 0.